src/include/cell.i

```diff
@@ -772,6 +857,14 @@ restart:
             break;
         flags = *p++; /* skip second descriptor byte */
 
+        if (LF_ISSET(WT_CELL_PREPARE))
+            F_SET(unpack, WT_CELL_UNPACK_PREPARE);
+        if (LF_ISSET(WT_CELL_TS_DURABLE_START))
+            WT_RET(__wt_vunpack_uint(
+              &p, end == NULL ? 0 : WT_PTRDIFF(end, p), &unpack->durable_start_ts));
+        if (LF_ISSET(WT_CELL_TS_DURABLE_STOP))
+            WT_RET(__wt_vunpack_uint(
+              &p, end == NULL ? 0 : WT_PTRDIFF(end, p), &unpack->durable_stop_ts));
```

Comment:
From looking at the code below, the other `stop` pairs are stored as a delta (i.e. we add the number to the `start` equivalent to get the real value). Do we not want this for prepare?

Reply:
You're right - I'm fixing this to store as a delta.  I'll also reorder the packing/unpacking slightly - I just noticed that all the start_* are grouped together, followed by the stop_*